bofire/data_models/domain/features.py

```diff
@@ -348,6 +350,16 @@ def _get_transform_info(
                     [f"{feat.key}{_CAT_SEP}{d}" for d in feat.descriptors]
                 )
                 counter += len(feat.descriptors)
+            elif isinstance(specs[feat.key], MolFeatures):
```

Comment:
Can you also include this in the tests?

Reply:
Added MolecularInput to test_inputs_get_transform_info